sources/packages/api/src/route-controllers/application/models/application.model.ts

```diff
@@ -0,0 +1,6 @@
+import { IsNotEmpty } from "class-validator";
+
+export class CreateApplicationDto {
+  @IsNotEmpty()
+  data: any;
```

Comment:
Can we have a Type here instead of any?

Reply:
This is part of the main idea to have dynamic forms generated using formio. This parameter should be able to receive anything sent by the client.
The next step that I am working on is to have it validated in the same way that happened on the client side.